<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="1" ignored="0" total="8" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="sbi-general automation test" duration-ms="7809" started-at="2020-09-14T15:04:47 IST" finished-at="2020-09-14T15:04:54 IST">
    <groups>
    </groups>
    <test name="sbi-general" duration-ms="7809" started-at="2020-09-14T15:04:47 IST" finished-at="2020-09-14T15:04:54 IST">
      <class name="sbi.testcases.tc01_logintest">
        <test-method status="PASS" signature="setup()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="setup" is-config="true" duration-ms="3447" started-at="2020-09-14T15:04:47 IST" finished-at="2020-09-14T15:04:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="tc01_login()[pri:1, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc01_login" duration-ms="2291" started-at="2020-09-14T15:04:51 IST" finished-at="2020-09-14T15:04:53 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc01_login -->
        <test-method status="PASS" signature="capturescreenshot(org.testng.ITestResult)[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="capturescreenshot" is-config="true" duration-ms="3" started-at="2020-09-14T15:04:53 IST" finished-at="2020-09-14T15:04:53 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc01_login status=SUCCESS method=tc01_logintest.tc01_login()[pri:1, instance:sbi.testcases.tc01_logintest@6ebc05a6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- capturescreenshot -->
        <test-method status="PASS" signature="tc02_enrollmenttab()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc02_enrollmenttab" duration-ms="1361" started-at="2020-09-14T15:04:53 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc01_login" finished-at="2020-09-14T15:04:54 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc02_enrollmenttab -->
        <test-method status="PASS" signature="capturescreenshot(org.testng.ITestResult)[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="capturescreenshot" is-config="true" duration-ms="0" started-at="2020-09-14T15:04:54 IST" finished-at="2020-09-14T15:04:54 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc02_enrollmenttab status=SUCCESS method=tc01_logintest.tc02_enrollmenttab()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- capturescreenshot -->
        <test-method status="FAIL" signature="tc03_searchagent()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc03_searchagent" duration-ms="3" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc02_enrollmenttab" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/commons/lang3/RandomStringUtils]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/commons/lang3/RandomStringUtils
at sbi.testcases.basetestcase.randomnumeric(basetestcase.java:76)
at sbi.testcases.tc01_logintest.tc03_searchagent(tc01_logintest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.RandomStringUtils
at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
... 34 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc03_searchagent -->
        <test-method status="PASS" signature="capturescreenshot(org.testng.ITestResult)[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="capturescreenshot" is-config="true" duration-ms="134" started-at="2020-09-14T15:04:54 IST" finished-at="2020-09-14T15:04:54 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc03_searchagent status=FAILURE method=tc01_logintest.tc03_searchagent()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- capturescreenshot -->
        <test-method status="SKIP" signature="tc04_createagentoptions()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc04_createagentoptions" duration-ms="0" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc03_searchagent" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method tc01_logintest.tc04_createagentoptions()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method tc01_logintest.tc04_createagentoptions()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc04_createagentoptions -->
        <test-method status="SKIP" signature="tc05_createagent()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc05_createagent" duration-ms="0" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc04_createagentoptions" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method tc01_logintest.tc05_createagent()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method tc01_logintest.tc05_createagent()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc05_createagent -->
        <test-method status="SKIP" signature="tc06_savebasicdetails()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc06_savebasicdetails" duration-ms="0" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc05_createagent" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method tc01_logintest.tc06_savebasicdetails()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method tc01_logintest.tc06_savebasicdetails()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc06_savebasicdetails -->
        <test-method status="SKIP" signature="tc07_personalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc07_personalinformation" duration-ms="0" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc06_savebasicdetails" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method tc01_logintest.tc07_personalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method tc01_logintest.tc07_personalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc07_personalinformation -->
        <test-method status="SKIP" signature="tc08_savepersonalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6]" name="tc08_savepersonalinformation" duration-ms="0" started-at="2020-09-14T15:04:54 IST" depends-on-methods="sbi.testcases.tc01_logintest.tc07_personalinformation" finished-at="2020-09-14T15:04:54 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method tc01_logintest.tc08_savepersonalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method tc01_logintest.tc08_savepersonalinformation()[pri:0, instance:sbi.testcases.tc01_logintest@6ebc05a6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc08_savepersonalinformation -->
      </class> <!-- sbi.testcases.tc01_logintest -->
    </test> <!-- sbi-general -->
  </suite> <!-- sbi-general automation test -->
</testng-results>
